### PR TITLE
[test] Add x nodes y actors test to nightly tests

### PR DESCRIPTION
**Pull request by fishbone**
## Why are these changes needed?
Add test to run 500 nodes 10k actors. We can bump this up later.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/nightly_tests/scability_test/compute_config.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+cloud_id: cld_4F7k8814aZzGG8TNUGPKnc
+
+region: us-west-2
+
+aws:
+    BlockDeviceMappings:
```
Comment (rkooo567) [inline release/nightly_tests/scability_test/compute_config.yaml:6]: maybe unnecessary?

File: release/nightly_tests/scability_test/scability_test.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+
+def parse_script_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--cpus-per-actor", type=float, default=0.1)
+    parser.add_argument("--total-actors", type=int, default=20000)
+    return parser.parse_known_args()
+
+
+def main():
```
Comment (rkooo567) [inline release/nightly_tests/scability_test/scability_test.py:33]: Btw consider adding tests where you invoke Ctrl+C and start a new driver? Wdyt?

File: release/nightly_tests/nightly_tests.yaml [hunk 3]
```diff
@@ -316,3 +316,13 @@
     timeout: 7200
     prepare: python wait_cluster.py 32 1000
     script: python dask_on_ray/dask_on_ray_sort.py --nbytes 1_000_000_000_000 --npartitions 1000 --num-nodes 31 --ray --data-dir /tmp/ray --s3-bucket core-nightly-test
+
+- name: scability_test
```
Comment (rkooo567) [inline release/nightly_tests/nightly_tests.yaml:320]: Use a bit more concrete name? `large_scale_actor_scheduling_test`

**Review by rkooo567 — COMMENTED** [review]
Can you add this to `build_pipeline.py`? Also, what's the main difference from our scalability envelope?

**Comment by iycheng** [comment]
> Can you add this to `build_pipeline.py`? Also, what's the main difference from our scalability envelope?

Do you mean many actors?

I think this one focuses on a large number of nodes and each node only has a limited number of actors (20 per node).

**Comment by rkooo567** [comment]
Will merge once it is added to build_pipeline.py

**Review by rkooo567 — APPROVED** [review]
It's ready right?

**Pull request merged**